src/symbols/SymStore.h

```diff
@@ -40,10 +40,15 @@ class SymStore {
     SymStore(SymStore &&) = default;
     SymStore & operator=(SymStore &&) = default;
     // Constructs a new symbol.
-    SymRef newSymb(char const * fname, SRef rsort, vec<SRef> const & args, SymbolConfig const & symConfig);
+
+    SymRef newSymb(char const * fname, SRef rsort, vec<SRef> const & args, SymbolConfig const & symConfig,
+                   bool subSymb = false);
     SymRef newSymb(char const * fname, SRef rsort, vec<SRef> const & args) {
         return newSymb(fname, rsort, args, SymConf::Default);
     }
+    SymRef newUnparsableSymb(char const * fname, SRef rsort, vec<SRef> const & args, SymbolConfig const & symConfig) {
```

Comment:
We should come up with a better name.
`SymStore` does not care about parsing.

Reply:
Though it's also not that good, because this is the symbol which would be seen by the users, using the tool API 🤔